src/StatusQ/Controls/StatusSlider.qml

```diff
@@ -6,19 +6,28 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1
 
+/*!
+  /note beware, the slider processes the mouse events only in its control space. So it must be at least as high
+  as the /c handle so user can grab it fully.
+ */
 Slider {
     id: statusSlider
 
     implicitWidth: 360
-    implicitHeight: 4
+    implicitHeight: Math.max(handle.implicitHeight, background.implicitHeight)
 
     leftPadding: 0
 
     background: Rectangle {
         id: bgRect
+
+        x: statusSlider.leftPadding
+        y: statusSlider.topPadding + statusSlider.availableHeight / 2 - height / 2
         implicitWidth: 100
         implicitHeight: 4
+        width: statusSlider.availableWidth
```

Comment:
does it make sense to keep both `implicitWidth` and `width` set differently?

Reply:
Thanks for the heads up, I removed it.
I followed [Qt's example](https://doc.qt.io/qt-5/qtquickcontrols2-customize.html#customizing-slider), but I agree that it is not needed given the `Control`'s background requirements: `"Note: If the background item has no explicit size specified, it automatically follows the control's size. In most cases, there is no need to specify width or height for a background item."`